dependency-injection/2.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Dependency Injection 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Dependency Injection 2.0 (milestone)"
+date: 2020-05-18
```

Comment:
The date used here is the trigger date for making it public.  So, once this PR is merged, then this date will be used to determine when the page becomes public on the website.  May 18th is probably okay.  I don't think every component has to go public on the same date.  Just wanted you to be aware.